### ISSUE TITLE
Is it possible to build a x64 version?

### ISSUE THREAD

**Issue by qianwch** [sammoth/foo_mpv#9]
I am using foobar2000 wine64 version, as I am in a macOs box:)

**Comment by sammoth** [comment]
Hi, I won't be able to update for foobar 2 for a little while. It looks like someone has made a 64 bit build with the new SDK here though: https://github.com/intrigit/foo_mpv/releases/tag/v0.6.11-beta

**Comment by ghost** [comment]
Yeah, really needing this. The version above doesn't work with .mov files for some reason, whereas this x86 version did just fine.
